src/components/Banner/Banner.mdx

```diff
@@ -0,0 +1,184 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+
+import * as BannerStories from "./Banner.stories";
+import Link from "../Link/Link";
+import ComponentChangelogTable from "../../utils/ComponentChangelogTable";
+import { changelogData } from "./bannerChangelogData";
+
+<Meta of={BannerStories} />
+
+# Banner
+
+| Component Version | DS Version   |
+| ----------------- | ------------ |
+| Added             | `0.23.2`     |
+| Latest            | `Prerelease` |
+
+## Table of Contents
+
+- {<Link href="#overiew" target="_self">Overview</Link>}
+- {<Link href="#component-props" target="_self">Component Props</Link>}
+- {<Link href="#accessibility" target="_self">Accessibility</Link>}
+- {<Link href="#variants" target="_self">Variants</Link>}
+- {<Link href="#banner-heading" target="_self">Banner Heading</Link>}
+- {<Link href="#banner-icon" target="_self">Banner Icon</Link>}
+- {<Link href="#with-html-content" target="_self">With HTML Content</Link>}
+- {<Link href="#custom-color" target="_self">Custom Color</Link>}
+- {<Link href="#dismissible" target="_self">Dismissible</Link>}
+- {<Link href="#changelog" target="_self">Changelog</Link>}
+
+## Overview
+
+<Description of={BannerStories} />
+
+**IMPORTANT:** The `Banner` component mirrors the `Notification` component but
```

Comment:
do `Banner` and `Notification` have any functional or display differences? Also, when you say on top of the screen, does that mean above the DS header and or the local header?

Reply:
Sounds good. I took most of this, including the word `mirrors` from the spec doc so that may need an update as well.

Reply:
May also describe an example?

Reply:
@EdwinGuzman I updated the spec doc.

@oliviawongnyc I am going to stay away from specifics content examples.  I am hoping that the location distinctions will be enough to differentiate the intended uses.